src/generators/TypeScriptGenerator.ts

```diff
@@ -97,81 +117,178 @@ async function generateInterfaceFile(
   const interfaceWriter = new FileWriter(
     interfaceContents,
     interfacePath,
-    namespace,
+    null,
     `${normalizedName}.ts`
   );
   await interfaceWriter.writeFile();
 }
 
-async function generate(
-  entityDefinition: EntityDefinition,
-  baseDirectory: FilePath
-): Promise<string> {
-  await generateClassFile(entityDefinition, baseDirectory);
+async function generateIndexFiles(
+  entityCollection: EntityCollection
+): Promise<void> {
+  // Generate ResourceMapping.ts
+  const hierarchy = new TypeHierarchy(entityCollection);
+  const resourceType = DataType.getInstance("FHIR", "Resource", entityCollection.baseDir);
+  const resourceChildren: Array<DataType> = hierarchy.getAllChildrenFor(resourceType);
+
+  const resourceMapContents = resourceMapTemplate({
```

Comment:
Codacy found an issue: [Unsafe assignment of an any value.](https://app.codacy.com/gh/MeasureAuthoringTool/model-info-parser/pullRequest?prid=5923420)